src/components/TextField/TextField.test.tsx

```diff
@@ -48,3 +50,42 @@ describe('<TextField />', () => {
     });
   });
 });
+
+describe('TextField#focus', () => {
+  let ref: any;
+  const setRef = (i: any) => (ref = i);
+
+  describe('with default options', () => {
+    beforeEach(() => {
+      mount(<TextField componentRef={setRef} />);
+    });
+
+    it('focus', () => {
+      const focusSpy = jest.spyOn(FabricTextField.prototype, 'focus');
+      const setSelectionRangeSpy = jest.spyOn(FabricTextField.prototype, 'setSelectionRange');
```

Comment:
these 2 declarations are repeated in another block below. i'd declare them in the top-level describe block. then i'd make this `it` block a `describe` block which explains the verb/action it's taking, then nest the expectation in an `it` block. maybe something like this:

```
describe('with componentRef', () => {
  let ref: ITextField | undefined;
  let focusSpy: jest.Mock;
  let setSelectionRangeSpy: jest.Mock;

  const setRef = (node: ITextField) => (ref = node);

  describe('when focused', () => {
    const focusSpy = jest.spyOn(FabricTextField.prototype, 'focus');
    ref.focus();
  });

  it('focuses the text input', () => {
    expect(focusSpy).toHaveBeenCalled();
  });

  // Multiple expectations should be out here in their own `it` block
});
```

Reply:
Yup, looks good.  I'll do that.

Reply:
done